### ISSUE TITLE
No visual difference between differing folders

### ISSUE THREAD

**Issue by bachowny** [yousseb/meld#83]
I'm trying to compare two folders with thousands of files and deep nesting, but Meld doesn't show me any indication of which folders are different. This unfortunately renders it unacceptable as a replacement for BeyondCompare for me. I do like the look and feel of Meld though and thought the feedback might be useful.

**Comment by bachowny** [comment]
![image](https://user-images.githubusercontent.com/36650922/53990449-6ce1d380-417c-11e9-9a5d-fac0413d994a.png)

**Comment by md1023** [comment]
You may have to wait until python processes the result. Python slowness is one of the reasons I've installed Linux on work Macbook.

**Comment by jslvtr** [comment]
Out of curiosity, would you expect the folder to also be blue or something to show that something inside the folder is different?

Meld expands folders which have differences and collapses those that don't, as its way of showing this.

**Comment by bachowny** [comment]
Yes I would expect that the folder show some indication of differences inside. The way that Meld expands the folders with differences was not useful when I have 10,000 files in 3,000 folders to compare.
